diff -Naur bacula-9.0.0.old/src/dird/bacula-dir.conf.in bacula-9.0.0/src/dird/bacula-dir.conf.in
--- bacula-9.0.0.old/src/dird/bacula-dir.conf.in	2017-07-10 08:52:38.930834509 +0200
+++ bacula-9.0.0/src/dird/bacula-dir.conf.in	2017-07-10 08:53:11.189444548 +0200
@@ -18,7 +18,7 @@
 Director {                            # define myself
   Name = @basename@-dir
   DIRport = @dir_port@                # where we listen for UA connections
-  QueryFile = "@scriptdir@/query.sql"
+  QueryFile = "@sysconfdir@/query.sql"
   WorkingDirectory = "@working_dir@"
   PidDirectory = "@piddir@"
   Maximum Concurrent Jobs = 20
diff -Naur bacula-9.0.0.old/src/dird/Makefile.in bacula-9.0.0/src/dird/Makefile.in
--- bacula-9.0.0.old/src/dird/Makefile.in	2017-07-10 08:52:38.930834509 +0200
+++ bacula-9.0.0/src/dird/Makefile.in	2017-07-10 08:54:24.363828341 +0200
@@ -124,11 +124,11 @@
 	   @if test "x${dir_group}" != "x" -a "x${DESTDIR}" = "x" ; then \
 	      chgrp -f ${dir_group} ${DESTDIR}${sysconfdir}/$$destconf ; \
 	   fi
-	@if test -f ${DESTDIR}${scriptdir}/query.sql; then \
+	@if test -f ${DESTDIR}${sysconfdir}/query.sql; then \
 	   echo "  ==> Saving existing query.sql to query.sql.old"; \
-	   $(MV) -f ${DESTDIR}${scriptdir}/query.sql ${DESTDIR}${scriptdir}/query.sql.old; \
+	   $(MV) -f ${DESTDIR}${sysconfdir}/query.sql ${DESTDIR}${sysconfdir}/query.sql.old; \
 	fi
-	${INSTALL_DATA} query.sql ${DESTDIR}${scriptdir}/query.sql
+	${INSTALL_DATA} query.sql ${DESTDIR}${sysconfdir}/query.sql
 	@if test -f static-bacula-dir; then \
 	   $(LIBTOOL_INSTALL) $(INSTALL_PROGRAM) static-bacula-dir $(DESTDIR)$(sbindir)/static-bacula-dir; \
 	fi
@@ -137,7 +137,7 @@
 uninstall:
 	(cd $(DESTDIR)$(sbindir); $(RMF) bacula-dir bdirjson)
 	(cd $(DESTDIR)$(sysconfdir); $(RMF) bacula-dir.conf bacula-dir.conf.new)
-	(cd $(DESTDIR)$(scriptdir); $(RMF) query.sql)
+	(cd $(DESTDIR)$(sysconfdir); $(RMF) query.sql)
 
 
 
